frontend/src/Components/Styles/index.scss

```diff
@@ -0,0 +1,48 @@
+body{
+    background: lighten($color: #dd7474, $amount: 30);
+    text-align: center;
+}
+.title-bar{
+    padding-top: 2%;
+    padding-bottom: 2%;
+    background:lighten($color: #51ff9a, $amount: 30);
+    border-style: round($number: 40);
+}
+
+input[type=submit] {
+    background-color: lighten($color: #07ff6e, $amount: 30);
+    color: white;
+    padding: 10px 14px;
+    margin: 8px 0;
+    border: none;
+    border-radius: 4px;
+    cursor: pointer;
+}
+.container{
+    margin-top: 5%;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: rgb(53, 99, 70);
+}
+
+li {
+    float: left;
+}
+
+li a {
```

Comment:
Never use nested selectors. It's against BEM. Styles must not depend on markup, specific tags. Styles must depend on meaningful classes. But styles don't know where this class is.